.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "externals/wepoll"]
```

Comment:
Is there a way to do this without submodules? I'd really like to avoid bringing in a submodule dependency here, as they've caused me a lot of problems in the past (e.g. `git archive` doesn't work anymore, so GitHub download links don't work).

Looking at how [tiny wepoll is](https://github.com/piscisaureus/wepoll), maybe vendoring it with [git subtree](https://www.atlassian.com/git/tutorials/git-subtree) is the way to go.

Reply:
works for me.  just updated accordingly.